adoc/Release-Notes.adoc

```diff
@@ -119,7 +119,7 @@ Note that the external database server *must* support both TLS and non-encrypted
 
 * {product} 1.5.2 has been tested successfully with external database instances hosted on Amazon RDS MariaDB and Azure MariaDB. Amazon Aurora, Amazon RDS MySQL or Azure MYSQL are not yet supported configurations.  
```

Comment:
Should we account for the 2nd sentence as to why? I'd lean towards just stating what we successfully tested in the release notes.

Reply:
Yeah, we can drop the 2nd sentence. Should we add "Any other external database setup is not supported at this point."?

Reply:
2nd sentence removed. Awaiting more feedback from @troytop